.gitignore

```diff
@@ -1,12 +1,10 @@
-!*.json
 *.pyc
 dist/
 build/
 connections.config
 sqlite.db*
-scripts/
 retriever.egg-info
 apidocs
 .DS_Store
 .cache/*
-*.bak
+scripts/*.py
```

Comment:
I would think that it is good to block the entire scripts folder, unless otherwise